tests/scripts/icmp_responder.py

```diff
@@ -55,8 +62,10 @@ def __call__(self):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="ICMP responder")
     parser.add_argument("--intf", "-i", dest="ifaces", required=True, action="append", help="interface to listen for ICMP request")
+    parser.add_argument("--dst_mac", "-m", dest="dst_mac", required=False, action="store", help="The source MAC to use for ICMP replies")
```

Comment:
Shouldn't the description be "The destination MAC to be used in ICMP replies"?

Reply:
Ack, thanks